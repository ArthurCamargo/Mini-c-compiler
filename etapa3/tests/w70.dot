digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n1 -> n4;
n2;
n2 [label="10"];
n3;
n3 [label="b"];
n4;
n4 -> n5;
n4 -> n6;
n5;
n5 [label="b"];
n6;
n6 [label="a"];
n4 [label="<="];
n1 [label="<="];
n0 [label="f"];

}