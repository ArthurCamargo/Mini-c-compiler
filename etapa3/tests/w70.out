0x55e1b94d9350
0x55e1b94d9350, 0x55e1b94d9480
0x55e1b94d9480
0x55e1b94d9480, 0x55e1b94d9430
0x55e1b94d9480, 0x55e1b94d93c0
0x55e1b94d9480, 0x55e1b94d95d0
0x55e1b94d9430
0x55e1b94d9430 [label="10"];
0x55e1b94d93c0
0x55e1b94d93c0 [label="b"];
0x55e1b94d95d0
0x55e1b94d95d0, 0x55e1b94d9580
0x55e1b94d95d0, 0x55e1b94d94f0
0x55e1b94d9580
0x55e1b94d9580 [label="b"];
0x55e1b94d94f0
0x55e1b94d94f0 [label="a"];
0x55e1b94d95d0 [label="<="];
0x55e1b94d9480 [label="<="];
0x55e1b94d9350 [label="f"];
